stdpopsim/species.py

```diff
@@ -236,3 +243,27 @@ def get_genetic_map(self, id):
             if gm.id == id:
                 return gm
         raise ValueError(f"Genetic map '{self.id}/{id}' not in catalog")
+
+    def add_annotations(self, annotations):
+        if annotations.id in [an.id for an in self.annotations]:
+            raise ValueError(
+                    f"Annotations '{self.id}/{annotations.id}' "
+                    "already in catalog.")
+        annotations.species = self
+        self.annotations.append(annotations)
+
+    def get_annotations(self, id):
+        """
+        Returns a set of annotations with the specified ``id``.
+
+        :param str id: The string identifier for the set of annotations
+            A complete list of IDs for each species can be found in the
+            "Annotations" subsection for the species in the :ref:`sec_catalog`.
```

Comment:
So we'll need some cleverness added to `docs/_ext/speciescatalog.py` in order to get the annotation id's listed in the catalog docs. (just mentioning this so we remember to open a ticket when this is merged)